src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -457,7 +457,7 @@ private void SetNodeInfo(CustomNodeInfo newInfo)
             }
 
             CustomNodeInfo info;
-            if (NodeInfos.TryGetValue(newInfo.FunctionId, out info))
+            if (!isTestMode && NodeInfos.TryGetValue(newInfo.FunctionId, out info))
```

Comment:
I'm a bit confused - if you use testMode to turn off the behavior you just added how do you test it?

Reply:
I have added a separate test to test this functionality.